src/main/java/org/fundacionjala/coding/omar/encryption/VigenereEncryption.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * @(#)VigenereEncryption.java
+ *
+ * Copyright (c) 2018 Jala Foundation.
+ * Address
+ * All rights reserved.
+ *
+ * This software is the confidential and propietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Sun.
+ */
+package org.fundacionjala.coding.omar.encryption;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * @author Omar Limbert Huanca Sanchez - AT-[06].
+ * @version 1.0.
+ */
+public class VigenereEncryption extends Encryption {
+
+    private int indexVigenereKey;
+
+    /**
+     * Constructor to initialize index for Vigenere Key.
+     */
+    public VigenereEncryption() {
+        super();
+        indexVigenereKey = 0;
```

Comment:
this line of code is redundant since integer attributes of class are by default zero